examples/DotNetCore/EventHub/EventHubConnection.cs

```diff
@@ -0,0 +1,17 @@
+namespace SimpleAppConfigEventHub
+{
+    public class EventHubConnection
+    {
+        public string EventHubConnectionString { get; set; }
+
+        public string EventHubName { get; set; }
+
+        public string BlobStorageConnectionString { get; set; }
+
+        public string BlobStorageContainerName { get; set; }
+
+        public string EventHubConsumerGroup { get; set; }
```

Comment:
>EventHubConsumerGroup [](start = 22, length = 21)

Will the usage different consumer groups affect the applications event processing? For example, will it only see a subset of total events?

Reply:
Each consumer group would get a copy of the event. Consumer groups are the way for user applications to run multiple instances. The number of consumer groups will need to be the same as the number of application instances for each instance to get the notification.